tests/e2e/pageobjects/ide/QuickOpenContainer.ts

```diff
@@ -36,7 +36,7 @@ export class QuickOpenContainer {
     public async clickOnContainerItem(itemText: string, timeout: number = TimeoutConstants.TS_SELENIUM_TOP_MENU_QUICK_CONTAINER_TIMEOUT) {
         Logger.debug(`QuickOpenContainer.clickOnContainerItem "${itemText}"`);
 
-        const quickContainerItemLocator: By = By.css(`div[aria-label="${itemText}"]`);
```

Comment:
I am not sure about this one, have it works for all cases? Because I also observed one case when container has diferent  locator but in other case they are not. I workarounded that by: https://github.com/eclipse/che/blob/main/tests/e2e/tests/plugins/GitHubPullRequestPlugin.spec.ts#L94-L95

Reply:
Looks like it may be a similar case
![Screenshot from 2021-10-01 10-26-57](https://user-images.githubusercontent.com/22724619/135581968-9b6b28db-756c-41f7-9c79-c46853916534.png)
e

Reply:
@Ohrimenko1988 not sure I understand how to implement this workaround for Terminal->Run Task use

Reply:
The issue here is that https://github.com/eclipse/che/blob/447e0415def3c3689d57264b5765e37a21e43714/tests/e2e/pageobjects/ide/TopMenu.ts#L61
fails to click on the element because of a `dom` change

Reply:
After discussion, changed the approach and opened a new issue for fix of QuickOpenContainers https://github.com/eclipse/che/issues/20568